compiler-rt/test/msan/prctl.cpp

```diff
@@ -0,0 +1,24 @@
+// RUN: %clangxx_msan -O0 %s -o %t && not %run %t 2>&1 | FileCheck %s
+// REQUIRES: linux
+
+#include <linux/prctl.h>
+#include <sys/prctl.h>
+
+int main(void) {
+  prctl(PR_SET_NAME, "tname");
```

Comment:
this test is good

but because the PR changes `compiler-rt/lib/sanitizer_common/`
we need a test in `compiler-rt/test/sanitizer_common/`

It should be set/get/strcmp/assert